magkit-test-server/src/main/java/de/ibmix/magkit/test/server/MagnoliaTomcatExtension.java

```diff
@@ -281,6 +284,9 @@ public void lifecycleEvent(LifecycleEvent event) {
                 throw new RuntimeException("Tomcat didn't come up after " + waitedMillis + "ms?!");
             }
         }
+        // Needed since Magnolia 6.3.x for GET-requests to magnolia: init ServerConfiguration: instanceId;
```

Comment:
Weird, let's keep it like this for the moment, can you please create a ticket as a reminder for me to research this when I have time?